crates/re_renderer/src/renderer/rectangles.rs

```diff
@@ -435,17 +464,17 @@ impl RectangleDrawData {
                 bind_group: ctx.gpu_resources.bind_groups.alloc(
                     &ctx.device,
                     &ctx.gpu_resources,
-                    &BindGroupDesc {
+                    &(BindGroupDesc {
```

Comment:
lot of unnecessary paranthesis that slipped in this file, actually not sure why the formatter doesn't remove them again. Any reason you added all of these?